src/NuGetTrends.Api/Startup.cs

```diff
@@ -91,5 +107,19 @@ public void Configure(IApplicationBuilder app)
             app.UseSwagger();
             app.UseMvc();
         }
+
+        class SetUserIpEventProcessor : ISentryEventProcessor
+        {
+            private readonly IHttpContextAccessor _accessor;
+
+            public SetUserIpEventProcessor(IHttpContextAccessor accessor) => _accessor = accessor;
+
+            public SentryEvent Process(SentryEvent @event)
+            {
+                // Will take the user IP considering the x-forwarded of the proxies
+                @event.User.IpAddress = _accessor.HttpContext.Connection.RemoteIpAddress.ToString();
```

Comment:
We might have to think about this.. due to GDPR?

Reply:
I'm testing this out. I'd put this into the Sentry SDK instead if it makes sense to always send the remote IP.
I'm anyway still getting the proxy IP instead of the right IP